src/plugins/intel_cpu/src/mkldnn_exec_network.cpp

```diff
@@ -505,12 +503,10 @@ bool MKLDNNExecNetwork::CanProcessDynBatch(const InferenceEngine::CNNNetwork &ne
         auto type = TypeFromName(op->get_type_name());
         if (type == Tile) {
             const auto tile = std::dynamic_pointer_cast<const ngraph::opset1::Tile>(op);
```

Comment:
Could you please move `tile` initialization closer to its usage in the `if` statement?